src/backend/distributed/sql/udfs/citus_job_status/latest.sql

```diff
@@ -0,0 +1,86 @@
+CREATE OR REPLACE FUNCTION pg_catalog.citus_job_status (
+    job_id bigint DEFAULT NULL,
+    raw boolean DEFAULT FALSE
+)
```

Comment:
No need to support NULL arguments I think.
```suggestion
) STRICT
```

Reply:
It was already strict, see several lines below here.